spec/integration/subscriber_lists_controller_spec.rb

```diff
@@ -55,6 +55,22 @@
         end
       end
 
+      context "creating subscriber list with a list_group_id" do
+        let!(:list_group_id) { SecureRandom.uuid }
```

Comment:
this doesn't need to have a bang: https://relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let

Besides that I don't think this even should be a let since we just use it in the one test, it'd be easier to follow if it was specified in the test block itself.

Reply:
Remove the let and just declared as a standard variable.